src/Reflection/EnumMethodReflection.php

```diff
@@ -67,8 +51,13 @@ public function isPublic(): bool
         return true;
     }
 
-    public function getPrototype(): MethodReflection
+    public function getPrototype(): ClassMemberReflection
     {
         return $this;
     }
+
+    public function getVariants(): array
+    {
+        return [];
```

Comment:
This does not seem right - you should return at least a single item. Can be a `FunctionVariant` instance.

Reply:
Because you have to report the return type of this method: `new ObjectType($this->classReflection->getName(), false)`.